Tools/simulation/gz/models/ssrc_holybro_x500/model_hitl.sdf

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sdf version='1.9'>
+  <model name='ssrc_holybro_x500_hitl'>
+    <include merge='true'>
+      <uri>model://ssrc_holybro_x500</uri>
+    </include>
+    <plugin filename="libmavlink_hitl_gazebosim.so" name="mavlink_interface::GazeboMavlinkInterface">
+      <robotNamespace/>
+      <imuSubTopic>/link/base_link/sensor/imu_sensor/imu</imuSubTopic>
+      <poseSubTopic>/pose/info</poseSubTopic>
+      <mavlink_addr>192.168.200.101</mavlink_addr>
+      <mavlink_udp_local_port>14542</mavlink_udp_local_port>
```

Comment:
Are these defined some script to start udp mavlink automatically? Or is it meant to be so that mavlink instance is configured manually via parameters or started from command line?

There are some SD-card scripts in px4-firmware/ssrc_config folder to configure HITL mavlink instance to start automatically.

Reply:
Yes, there are. For example this is for salukiV2 https://github.com/tiiuae/saluki-v2/blob/42cbe653acda38e66acd6e4ee7c6da7ca38930f4/init/rc.board_mavlink
There is same for salukiV3
https://github.com/tiiuae/saluki-v3/blob/a0cd5b9119e392e7d9c910a48df91ce05cbc6f36/init/rc.board_mavlink